tests/components/glances/test_sensor.py

```diff
@@ -28,3 +35,28 @@ async def test_sensor_states(
         assert hass.states.get(entity_entry.entity_id) == snapshot(
             name=f"{entity_entry.entity_id}-state"
         )
+
+
+async def test_uptime_variation(
+    hass: HomeAssistant, entity_registry: er.EntityRegistry
+) -> None:
+    """Test uptime small variation update."""
+
+    entry = MockConfigEntry(domain=DOMAIN, data=MOCK_USER_INPUT, entry_id="test")
+    entry.add_to_hass(hass)
+
+    # Setup with expected timestamp for reference
+    with freeze_time(MOCK_REFERENCE_DATE):
+        assert await hass.config_entries.async_setup(entry.entry_id)
+        await hass.async_block_till_done()
+
+    uptime_state = hass.states.get("sensor.0_0_0_0_uptime").state
+
+    coordinator: GlancesDataUpdateCoordinator = hass.data[DOMAIN]["test"]
```

Comment:
```suggestion
```
We should move time instead of using coordinator internals.

Search for `freezer.tick` to see examples.

Reply:
Thanks, that's what I was looking for